src/main/java/com/google/longrunning/OperationsSettings.java

```diff
@@ -66,6 +66,9 @@
   /** The default port of the service. */
   private static final int DEFAULT_SERVICE_PORT = 443;
 
+  private static final String DEFAULT_GENERATOR_NAME = "gapic";
+  private static final String DEFAULT_GENERATOR_VERSION = "0.0.5";
```

Comment:
@shinfan brought to my attention that we still want the version of `google-cloud` even when we have a GAPIC-only client, and storing that in `gapic` is the only way to get it under our current model. So, we'll have to go with that.